lib/tangocard/raas.rb

```diff
@@ -71,13 +69,9 @@ def self.delete_credit_card(params)
   #
   # Arguments:
   #   none
-  def self.rewards_index
-    if Tangocard.configuration.use_cache
-      clear_cache! if cache_expired?
-
-      @@rewards_response ||= Tangocard::Response.new(get_request('/rewards'))
-      @@rewards_response_expires_at = (Time.now.to_i + Tangocard.configuration.cache_ttl) if cache_expired?
-      @@rewards_response
+  def self.rewards_index(bypass_cache: false)
+    if Tangocard.configuration.use_cache && !bypass_cache
```

Comment:
would this read better if you flipped the logic to change `bypass_cache` to `use_cache`? you use `use_cache` for the configuration setting.